dandi/models.py

```diff
@@ -826,6 +826,7 @@ class BareAssetMeta(CommonModel):
         title="File Encoding Format", nskey="schema"
     )
     digest: Digest = Field(nskey="dandi")
+    dateModified: Optional[datetime] = Field(nskey="dandi")
```

Comment:
corresponds to https://schema.org/dateModified

Reply:
How do I indicate that?

Reply:
```suggestion
    dateModified: Optional[datetime] = Field(nskey="dandi", nskey="schema")
```
AFAIK

Reply:
may be adding a title right away which would also disambiguate from modifications to metadata...
```suggestion
    dateModified: Optional[datetime] = Field(title="Asset file modification date and time", nskey="dandi")
```
but that actually brings a question (@satra) either we should have dedicated `blobDateModified` and then `dateModified` as for when the entire assetMeta is modified (e.g. metadata entries added etc), which would correspond to the one in DB or when we extract metadata if locally.

Reply:
you can add a dandi specific `blobDateModified`

Reply:
Thanks @satra for the blessing.  @jwodder - lets add both 
- `dateModified ... (title="Asset (file or metadata) modification date and time", nskey="schema")` 
- `blobDateModified ... (title="Asset file modification date and time", nskey="dandi")` 

for the client blobDateModified -- of the file.  dateModified -- current time as of extraction of metadata (with a warning if file modification is past current time)

Reply:
Done.